.editorconfig

```diff
@@ -0,0 +1,30 @@
+# Manage common editor configurations
+# set for example indentation tab and space
```

Comment:
I find the second line confusing, what about - "for example, set indentation character"